### ISSUE TITLE
Incorrect methods listed in error message from checker

### ISSUE THREAD

**Issue by msridhar** [kelloggm/object-construction-checker#99]
Test case:
```java
@AutoValue
abstract class SetInsideBuild {
  public abstract String name();
  public abstract int size();
  static Builder builder() {
    return new AutoValue_SetInsideBuild.Builder();
  }
  @AutoValue.Builder
  abstract static class Builder {
    abstract Builder setName(String name);
    abstract Builder setSize(int value);
    abstract SetInsideBuild autoBuild();
    public SetInsideBuild build(Builder this) {
      return this.setSize(4).autoBuild(); // error here
    }
  }
}
```
I would expect an error on the marked line indicating that `setName()` may not have been called.  Instead, the tool reports:
```
error: [finalizer.invocation.invalid] This finalizer cannot be invoked, because the following methods have not been called: setName() setSize() 
      return this.setSize(4).autoBuild();
```
This is pretty confusing 😄 I confirmed that I get no error message if I add an annotation to fix `build()`:
```java
    public SetInsideBuild build(@CalledMethods("setName") Builder this) {
      return this.setSize(4).autoBuild();
    }
```
So I'm pretty sure this is an issue in printing the error message.  I'm not sure how to write a test for the exact message using the Checker Framework testing tools.
